auth0/v3/rest.py

```diff
@@ -64,7 +64,7 @@ class RestClient(object):
             (defaults to 3)
     """
 
-    def __init__(self, jwt, telemetry=True, timeout=5.0, options=None):
+    def __init__(self, jwt, telemetry=True, timeout=5.0, options=None, pool_connection=10, pool_maxsize=10):
```

Comment:
You need to add `pool_connections` and `pool_maxsize` to `RestClientOptions` and then access them through the `options` kwarg. (this should fix the tests as well)

And, can you normalise the name of this to `pool_connections` everywhere